src/resqml2_0_1/FluidBoundaryFeature.cpp

```diff
@@ -39,4 +41,14 @@ FluidBoundaryFeature::FluidBoundaryFeature(soap* soapContext, const string & gui
 	setMetadata(guid, title, std::string(), -1, std::string(), std::string(), -1, std::string());
 }
 
+std::vector<epc::Relationship> FluidBoundaryFeature::getAllEpcRelationships() const
+{
+	vector<Relationship> result = BoundaryFeature::getAllEpcRelationships();
+
```

Comment:
The source rel with rock fluid unit feature should be listed here.
Ideally it should be done in BoundaryFeature superclass